backend/src/core/services/file-attachment.service.ts

```diff
@@ -43,19 +48,19 @@ const sanitizeFiles = async (
   if (!isAcceptedType) {
     await EmailMessageTransactional.update(
       {
-        errorCode: 'Error 400: Unsupported file type',
+        errorCode: UNSUPPORTED_FILE_TYPE_ERROR_CODE,
       },
       {
         where: { id: emailMessageTransactionalId },
       }
     )
     throw new UnsupportedFileTypeError()
   }
-  const isSafe = await virusScan(files)
+  const isSafe = await FileAttachmentService.areFilesSafe(files)
```

Comment:
This function is in this file, we don't have to `FileAttachmentService.`

Reply:
This is creating unnecessary depending on the `FileAttachmentService` object

Reply:
Ohh so the reason I wrapped it in the `FileAttachmentService` object is so that I can mock the value here:
<img width="762" alt="image" src="https://user-images.githubusercontent.com/67887489/197998066-4e8eec43-913e-460f-b5cd-3d8d72120c1c.png">

Is there some other way I can achieve this without creating this dependency 🤔

Reply:
> Ohh so the reason I wrapped it in the `FileAttachmentService` object is so that I can mock the value here: <img alt="image" width="762" src="https://user-images.githubusercontent.com/67887489/197998066-4e8eec43-913e-460f-b5cd-3d8d72120c1c.png">
> 
> Is there some other way I can achieve this without creating this dependency 🤔

No u shouldn't be exporting a private function just so that u can mock it - this is a really really bad reason to export, mock the external dependency that it relies on